tests/test_cli.py

```diff
@@ -140,3 +142,58 @@ def test_inject_column_sort(package_path, column_sort_arg):
 def test_version():
     result = runner.invoke(app, ["version"])
     assert result.exit_code == 0
+
+
+from click.testing import CliRunner
+from paracelsus.cli import app
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class Comments(Base):
+    __tablename__ = "comments"
+    id = Column(Integer, primary_key=True)
+    text = Column(String)
```

Comment:
Don't do this- use the existing test database, it works perfectly for this use case. 

Also imports should never be here, always at the top of the file.